src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -457,6 +457,65 @@
 short-summary: Updates the instance failover group.
 """
 
+helps['sql instance-pool'] = """
+type: group
+short-summary: Manage instance pools.
+"""
+
+helps['sql instance-pool create'] = """
+type: command
+short-summary: Create an instance pool.
+examples:
+  - name: Example to create an instance pool (include --no-wait in the end to get an asynchronous experience)
```

Comment:
Use active voice. e.g. Create an instance pool ...

Reply:
Fixed it